batch/batch_preprocessing/algorithm.py

```diff
@@ -1,38 +0,0 @@
-import plot
```

Comment:
Could you also see why this file content disappears? Theoretically this PR should only has name change

Reply:
it should be still there. 

![image](https://github.com/ncsa/standalone-smm-analytics/assets/12547573/1ed3a230-9033-4549-873a-bb54e6eca836)